src/__tests__/lib/command/smartthings-command.test.ts

```diff
@@ -0,0 +1,161 @@
+import log4js from 'log4js'
+
+import { CLIConfig, loadConfig } from '../../../lib/cli-config.js'
+import { smartThingsCommand } from '../../../lib/command/smartthings-command.js'
+import { buildDefaultLog4jsConfig, loadLog4jsConfig } from '../../../lib/log-utils.js'
+
+
+jest.mock('log4js')
+jest.mock('../../../lib/cli-config.js')
+jest.mock('../../../lib/log-utils.js')
+jest.mock('../../../lib//table-generator')
+
+
+describe('smartThingsCommand', () => {
+	const defaultLogConfig = { config: 'default' } as unknown as log4js.Configuration
+	const buildDefaultLog4jsConfigMock = jest.mocked(buildDefaultLog4jsConfig).mockReturnValue(defaultLogConfig)
+	const logConfig = { config: 'final' } as unknown as log4js.Configuration
+	const loadLog4jsConfigMock = jest.mocked(loadLog4jsConfig).mockReturnValue(logConfig)
+	const log4jsConfigureMock = jest.mocked(log4js.configure)
+	const traceMock = jest.fn()
+	const loggerMock = {
+		trace: traceMock,
+	} as unknown as log4js.Logger
```

Comment:
I'm curious why `loggerMock` got formatted as multi-line instead of a single line.  It looks structurally similar to me to the definition for `logConfig`, which is single-line.

Reply:
When I wrote it I thought I was going to add more than what I ended up with. I've condensed this back to one line to match the format of `logConfig`.